### DISCUSSION TITLE
What algorithm are you using under the hood?

### DISCUSSION THREAD

**Discussion by enricobottazzi** [inducer/pymetis#85] [Q&A]
Hi, what is the actual algorithm from METIS that you are using for graph partitioning as part of your wrapper? In the readme you mention that `it only wraps the most basic graph partitioning functionality`. What is the functionality that you are referring to?

Thanks

**Comment by inducer** [comment]
I'm not sure it's a good idea to duplicate the METIS docs, given that this is just a wrapper. It should be straightforward to figure which underlying METIS routine is being called (and with what parameters). At this point, the METIS docs should be considered authoritative as to what happens with those inputs.
